packages/kuma-gui/src/test-support/mocks/src/resources.ts

```diff
@@ -0,0 +1,44 @@
+import type { EndpointDependencies, MockResponder } from '@/test-support'
+export default ({ fake }: EndpointDependencies): MockResponder => (_req) => {
+  const policies = Array.from(fake.kuma.policyNames({ includeAll: true }))
+  const legacyPolicies = [
+    'CircuitBreaker',
+    'FaultInjection',
+    'HealthCheck',
+    'ProxyTemplate',
+    'RateLimit',
+    'Retry',
+    'Timeout',
+    'TrafficLog',
+    'TrafficPermission',
+    'TrafficRoute',
+    'TrafficTrace',
+    'VirtualOutbound',
+    'MeshGatewayRoute',
+  ]
+  
+  return {
+    headers: {},
+    body: {
+      resources: fake.kuma.resourceNames({ includeAll: true }).map((name) => {
+        const scope = fake.helpers.arrayElement(['Mesh', 'Global'])
+
+        return {
+          name,
+          scope,
+          readOnly: fake.datatype.boolean(),
+          path: ((_name) => `${_name.includes('Mesh') ? _name.toLocaleLowerCase() : _name.replace(/([a-z0-9])([A-Z])/g, '$1-$2').toLowerCase()}s`)(name),
+          singularDisplayName: name,
+          pluralDisplayName: `${name}s`,
+          includeInFederation: fake.datatype.boolean(),
+          ...([...legacyPolicies, ...policies].find((policy) => policy.includes(name) /* includes legacy policies */) && { policy: {
```

Comment:
In my eyes this looked a bit weird due to the placement of the comment just infront of the closing bracket, would you be ok moving the comment to the line above?

```suggestion
          // only for resources that are policy type ones, including legacy policy types
          ...([...legacyPolicies, ...policies].find((policy) => policy.includes(name)) && { policy: {
```

Something like that ☝️